config/owasp/suppressions.xml

```diff
@@ -264,6 +264,13 @@
     <cpe>^pkg:javascript/jquery@1.5.1</cpe>
     <cve>CVE-2011-4969</cve>
   </suppress>
+  <suppress>
+    <notes><![CDATA[
+   relates to pkg:maven/org.seleniumhq.selenium/htmlunit-driver@2.36.0
+   ]]></notes>
+    <cpe>cpe:2.3:a:htmlunit_project:htmlunit:2.36.0:*:*:*:*:*:*:*</cpe>
+    <cve>CVE-2020-5529</cve>
+  </suppress>
```

Comment:
why do we have serenity and selenium dependencies at all?
Please note that FW's latest release is 1.1.0, but it does not yet include file upload/download feature.
You need to use '1.1.1-RDM-7716.1' for a prerelease that contains the latest stable release + upload&download.
testCompile group: 'uk.gov.hmcts', name: 'befta-fw', version: '1.1.1-RDM-7716.1'